### PR TITLE
fix: Remove 'lts' tag from published images

### PR DESCRIPTION
**Pull request by helio-frota**

### CODE REVIEW

File: build/tag.js [hunk 1]
```diff
@@ -66,11 +66,12 @@ function processImageData (data) {
       if (imageData) {
         tagImage(imageData.Id.split(':')[1], image, version);
         if (releases[version].lts) {
-          tagImage(imageData.Id.split(':')[1], image, 'lts');
+          if (version == versions[versions.length - 2]) {
```
Comment (lance) [inline build/tag.js]: You should use `===` here. Surprised that linting didn't catch this. Hmm.

Comment (lance) [reply build/tag.js]: Won't this comparison always return the next to last result? What does that mean when Node.js 8.0 is released? Wouldn't v7 then end up being tagged as 'lts'? That wouldn't work. I think we need to find the largest version number where `releases[version].lts` is `true`.

Comment (helio-frota) [reply build/tag.js]: yah but this already inside of `if (releases[version].lts)` to check if is true

Comment (helio-frota) [reply build/tag.js]: hm but maybe it will not work...

**Pull request merged**